src/app/backend/replicasetdetail.go

```diff
@@ -46,6 +46,14 @@ type ReplicaSetDetail struct {
 
 	// Detailed information about Pods belonging to this Replica Set.
 	Pods []ReplicaSetPod `json:"pods"`
+
+	// Internal endpoints of all Kubernetes services have the same label selector as this Replica Set.
+	// Endpoint is DNS name merged with ports.
+	InternalEndpoints []string `json:"internalEndpoints"`
```

Comment:
With more information from service it is reasonable to create dedicated structure. We can create ServiceDetail and place all that in there.